docs/faq.md

```diff
@@ -57,7 +57,7 @@ We welcome community contributions of database connectors. If you're able to hel
 
 ### Can Metabase connect to Google Analytics, Salesforce, etc?
```

Comment:
[Technically this should be](http://english.stackexchange.com/questions/8382/when-etc-is-at-the-end-of-a-phrase-do-you-place-a-period-after-it)

> Can Metabase connect to Google Analytics, Salesforce, etc.?

😬